src/osd/PeeringState.cc

```diff
@@ -1402,6 +1402,26 @@ bool PeeringState::needs_backfill() const
   return false;
 }
 
+/**
+* Returns whether a particular object can be safely read on this replica
+*/
+bool PeeringState::can_serve_replica_read(const hobject_t &hoid)
+{
+  ceph_assert(!is_primary());
+  eversion_t min_last_complete_ondisk = get_min_last_complete_ondisk();
+  psdout(10) << __func__ << " oid: " << hoid
+             << "min last complete ondisk: " << min_last_complete_ondisk
```

Comment:
Isn't min_last_complete_ondisk already in the dout prefix as mlcod=?

Reply:
Yes, removed.